src/RootLinkDef.h

```diff
@@ -0,0 +1,12 @@
+#ifdef __CLING__
+
+  #pragma link off all globals;
+  #pragma link off all classes;
+  #pragma link off all functions;
+  #pragma link C++ nestedclasses;
+
+  #pragma link C++ class podio::ROOTFrameReader;
```

Comment:
Stupid question: Why does the Reader not need a `+`?

Reply:
Because it is a stupid oversight ;-)